_includes/landing-page.md

```diff
@@ -0,0 +1,23 @@
+<div class="mb-xl-5 bg-cover bg-cover__bg-early-3-1">
+    <div class="p-2 p-md-5">
+    <h1 class="m-0 text-white">{{ page.title }}</h1>
+    <p class="mt-0 pb-4 text-white">{{ page.summary }}</p>
+      <div class="row d-lg-flex mx-0">
+      <!-- cards loop -->
```

Comment:
```suggestion
      {% comment %} cards loop {% endcomment %}
```

I wanna avoid using HTML comments in the docs wherever possible, because the Jekyll parser still evaluates their contents.